doc/guides/kconfig/tips.rst

```diff
@@ -25,9 +25,13 @@ When deciding whether something belongs in Kconfig, it helps to distinguish
 between symbols that have prompts and symbols that don't.
 
 If a symbol has a prompt (e.g. ``bool "Enable foo"``), then the user can change
-the symbol's value in the ``menuconfig`` or ``guiconfig`` interface (or by
-manually editing configuration files). Therefore, only put a prompt on a symbol
-if it makes sense for the user to change its value.
+the symbol's value in the ``menuconfig`` or ``guiconfig`` interface (see
+:ref:`menuconfig`), or by manually editing configuration files. Therefore, only
+put a prompt on a symbol if it makes sense for the user to change its value.
+
+Symbols without prompts are also called *invisible* symbols, because they don't
```

Comment:
'hidden', is a better term, right? At least, this is what we have been using

Reply:
Yeah, looking around a bit, it's more common. Now uses *hidden* to refer to promptless symbols, and just mentions *invisible* when explaining that promptless stuff can be invisible too.